libs/components/pages/src/lib/modules/link-list-recently-accessed/link-list-recently-accessed.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, input } from '@angular/core';
+import { SkyI18nModule } from '@skyux/i18n';
+
+import { SkyActionHubRecentLinksResolvePipe } from '../action-hub/action-hub-recent-links-resolve.pipe';
+import { SkyRecentLinksInput } from '../action-hub/types/recent-links-input';
+import { SkyLinkListComponent } from '../link-list/link-list.component';
+
+@Component({
+  selector: 'sky-link-list-recently-accessed',
+  standalone: true,
+  imports: [
+    SkyActionHubRecentLinksResolvePipe,
+    SkyI18nModule,
```

Comment:
I think we need to import the `SkyPagesResourcesModule` module instead of `SkyI18nModule`.

Reply:
Good call.